### DISCUSSION TITLE
vant4 search 组件 输入内容，会出现2个清除

### DISCUSSION THREAD

**Discussion by JerryLiang1222** [youzan/vant#12143] [General]
![Snipaste_2023-08-01_18-06-29](https://github.com/youzan/vant/assets/106816899/a00cc428-576d-4dc8-8a64-96b961369e07)

**Comment by wjw-gavin** [comment]
看上去左边那个清除 icon 不是 vant 提供的，检查下是不是自己代码提供的

**Comment by songxiangya001** [comment]
我也遇见这个问题  请问是怎么解决了
